cmd/ipfs-cluster-ctl/main.go

```diff
@@ -614,10 +614,23 @@ with "sync".
 
 When the --local flag is passed, it will only fetch the status from the
 contacted cluster peer. By default, status will be fetched from all peers.
+
+When the --filter is passed, it will only fetch the peer information
+where status of the pin matches with the filter value.
+Valid filter values are tracker status types("pinned", "pin_error", "unpinning" etc),
+an alias of tracker status type (queued or error), comma separated list of
+tracker status type and/or it aliases ("error,pinning")
+On passing invalid filter value no status information will be shown
+List of tracker status types
+https://github.com/ipfs/ipfs-cluster/blob/319c41cbf195b0453b8d1987991280d3121bac93/api/types.go#L66
 `,
 			ArgsUsage: "[CID]",
 			Flags: []cli.Flag{
 				localFlag(),
+				cli.StringFlag{
+					Name:  "filter",
+					Usage: "Comma seperated list of type tracker status and their aliases",
```

Comment:
`Comma-separated list of filters.` The list of acceptable values are on the command help which is printed with the command a few lines over this.